src/services/users/index.js

```diff
@@ -171,9 +171,9 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	if (!user) {
 		throw customError(RefreshTokenIsInvalidError)
 	}
-
+	const body = { id: user.id, email: user.email }
```

Comment:
i think we mentioned so far as we were selecting the id and email in the query, we would not need to do this, you can simply pass the data as `{user}`